### PR TITLE
fix(pwa): add meta tags and update manifest to support progressive web app

### PR DESCRIPTION
**Pull request by anthony-liddle**
Add meta tags and update the manifest json to support progressive web apps.

Using https://web.dev/add-manifest/ and https://web.dev/add-manifest/ as a guide.

More work may need to be done here later, but this is a good starting point.

### CODE REVIEW

**Review by aaga — APPROVED** [review]

File: docs/public/site.webmanifest [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "",
-  "short_name": "",
+  "name": "Mozilla Social",
```
Comment (aaga) [inline docs/public/site.webmanifest:2]: Actually I think this is the wrong file. This webmanifest is for the dev docs

**Review by aaga — CHANGES_REQUESTED** [review]

File: docs/public/site.webmanifest (hunk 1 above)
Comment (anthony-liddle) [reply docs/public/site.webmanifest:2]: oooh good call, lemme find where this should be done...

Comment (aaga) [reply docs/public/site.webmanifest:2]: Digging into this more, it looks like Elk auto-generates a manifest file: https://github.com/MozillaSocial/elk/blob/main/config/pwa.ts

The generated file exists on prod: https://mozilla.social/manifest-en-US.webmanifest

So we need to figure out why, despite having the manifest, PWA isn't working on prod even though it is working on staging (according to @marcin-kozinski on Slack; I don't have Android to test this)

Comment (anthony-liddle) [reply docs/public/site.webmanifest]: I've removed the change to the manifest. It looks like the autogenerated one (defined with properties at `/modules/pwa/i18n.ts`) should cover it.

If you approve it we can get this merged in

**Pull request merged**